### DISCUSSION TITLE
How do you handle cash outflows toward investments?

### DISCUSSION THREAD

**Discussion by paolobasso99** [firefly-iii/firefly-iii#6132] [Firefly III] [answered]
I am trying to understand what would be the best way to handle cash outflows toward investments.

I know that **Firefly is not designed to track investments** but in some way, I need to track that I am missing some money from my checking accounts because they are now in investments, and recording them as "expenses" does not make much sense.

One possible solution is to use a "Investments" account, not included in the net worth, where I **transfer** money when performing an investment. Of course, the account balance will not be up to date and only the money I put in is recorded, not the actual present value.
I could even set up different accounts/categories/tags for the investments.

What is the best way to track cash outflows toward investments? How do you implement it?

**Comment by JC5** [comment, marked as answer]
Personally, I have an extra asset account and I make transfers, just like you describe.

**Comment by Restmon** [comment]
> I am trying to understand what would be the best way to handle cash outflows toward investments.
> 
> I know that **Firefly is not designed to track investments** but in some way, I need to track that I am missing some money from my checking accounts because they are now in investments, and recording them as "expenses" does not make much sense.
> 
> One possible solution is to use a "Investments" account, not included in the net worth, where I **transfer** money when performing an investment. Of course, the account balance will not be up to date and only the money I put in is recorded, not the actual present value. I could even set up different accounts/categories/tags for the investments.
> 
> What is the best way to track cash outflows toward investments? How do you implement it?

If you use an "Investments" account and not include it in the net worth, wouldn't it make your net worth seem short ?
